src/Rel8/Table/Aggregate.hs

```diff
@@ -53,16 +61,18 @@ groupBy (toColumns -> exprs) = fromColumns $ htabulate $ \field ->
 --   items <- aggregate $ listAgg <$> itemsFromOrder order
 --   return (order, items)
 -- @
-listAgg :: Table Expr exprs => exprs -> Aggregate (ListTable exprs)
-listAgg (toColumns -> exprs) = fromColumns $
-  hvectorize
-    (\_ (Identity (DB a)) -> Aggregation $ listAggExpr a)
-    (pure exprs)
+listAgg :: Table Expr exprs => (i -> exprs) -> Aggregator i (ListTable exprs)
+listAgg f = lmap (toColumns . f) $ fromColumns $ HVectorize $ htabulate $ \(HMapTableField field) ->
```

Comment:
Might be worth doing this in `HVectorize.hs` itself? Basically to use `listAggExpr`, we need to know that `a ~ [x]`, which we get from pattern matching on `SSpec` from `hfield hspecs field`, but only when `field` comes from `HMapTableField`.